components/LiteMintNFT.vue

```diff
@@ -97,19 +98,34 @@ const formState = reactive({
   maxSupply: undefined
 })
 
-const formRef = ref()
 const { LCD_URL } = useRuntimeConfig().public
+const emit = defineEmits(['submit'])
 
-const isFormValid = computed(() => {
-  return formRef.value?.validate(formState).length === 0
+const formError = computed(() => {
+  const requiredFields = {
+    prefix: !!formState.prefix,
+    mintCount: !!formState.mintCount,
+    imageUrl: !!formState.imageUrl
+  }
+
+  return Object.entries(requiredFields)
+    .find(([_, isValid]) => !isValid)?.[0]?.toUpperCase() || ''
 })
 
+const isFormValid = defineModel<boolean>('valid')
```

Comment:
Sorry for changing back-and-forth, but since `defineModel` allow parent to modify this variable, maybe we should just stick with `ref` + `watch` + `emit`

Reply:
[♻️ Use emit event instead of v-model](https://github.com/likecoin/nft-book-press/pull/344/commits/8cb618ad0a9826cf4898535f2c8e1184f314f7e8)